contracts/ProtocolReserve/XVSVaultTreasury.sol

```diff
@@ -83,6 +86,29 @@ contract XVSVaultTreasury is AccessControlledV8, ReentrancyGuardUpgradeable {
         emit FundsTransferredToXVSStore(xvsStore, amountMantissa);
     }
 
+    /// @notice Function to sweep tokens from the contract
+    /// @param tokenAddress Address of the asset(token)
+    /// @param to Address to which assets will be transferred
+    /// @param amount Amount need to sweep for the contract
```

Comment:
```suggestion
    /// @param amount Amount need to sweep from the contract
```

Reply:
I see that you have deployed the new implementation with this new function, So, you can also ignore this change.